features/step_definitions/logging.rb

```diff
@@ -340,19 +340,27 @@
   if cluster_logging("instance").exists?
     @result = admin.cli_exec(:delete, object_type: 'clusterlogging', object_name_or_id: 'instance', n: 'openshift-logging')
     raise "Unable to delete instance" unless @result[:success]
-  end
-  step %Q/I wait for the resource "deployment" named "kibana" to disappear/
-  step %Q/I wait for the resource "elasticsearch" named "elasticsearch" to disappear/
-  step %Q/I wait for the resource "cronjob" named "curator" to disappear/
-  step %Q/I wait for the resource "daemonset" named "fluentd" to disappear/
-  step %Q/all existing pods die with labels:/, table(%{
-    | component=elasticsearch |
-  })
-  step %Q/all existing pods die with labels:/, table(%{
-    | component=kibana |
-  })
-  if BushSlicer::PersistentVolumeClaim.list(user: user, project: project).count > 0
-    admin.cli_exec(:delete, object_type: 'pvc', l: 'logging-cluster=elasticsearch', n: 'openshift-logging')
+    unless cluster_logging("instance").log_store_spec(cached: true).nil?
+      step %Q/I wait for the resource "elasticsearch" named "elasticsearch" to disappear/
+      success = wait_for(180, interval: 10) {
```

Comment:
@QiaolingTang why do you `wait` again here?  The previous step does that already, correct?

Reply:
@pruan-rht no, the previous step is to wait for CR elasticsearch/elasticsearch to disappear, these steps wait for the deployments which have label `cluster-name=elasticsearch` to disappear. 
I had hit an issue: the elasticsearch/elasticsearch was deleted, but the ES pods were not. So here I want to add these steps to check if the deployments can be removed or not, since the deployment names are dynamic, so I use label here.